core/project.go

```diff
@@ -169,3 +177,22 @@ func (t *Timetrace) editorFromEnvironment() string {
 
 	return defaultEditor
 }
+
+func (t *Timetrace) parentExists(project Project) error {
```

Comment:
I'd expect a function `parentExists` to return `(bool, error)`. If you want to just return an error, I suggest renaming this function to `assertParent`.